docs/postmaster.md

```diff
@@ -0,0 +1,223 @@
+# Using postmaster
```

Comment:
Im totally sure we need to stick to generic naming.
As soon, as now it becomes one project.
In future it will cause misunderstanding.